### DISCUSSION TITLE
actions/checkout@v3 "Receiving objects" less than 80 KiB/s

### DISCUSSION THREAD

**Discussion by jmglsn** [community/community#43834] [Actions] [answered]
### checkout action super slow bandwidth

Bug

### Body

Hey GitHub team,

we noticed that more and more of our jobs fails as actions/checkout@v3 checkout runs super slow. As an example, all related jobs face a bandwidth that looks like this: `Receiving objects:  16% (2055/12840), 708.00 KiB | 70.00 KiB/s` .

How can we speed this up again?

Thank you!

**Comment by joelnotified** [comment]
Can confirm that we are seeing the same issue across multiple repositories in our organization. Not sure if it's some kind of throttling issue or something broken on the GitHub side?

- Getting errors like this: `Failed to download action 'https://api.github.com/repos/Azure/login/tarball/92a5484dfaf04ca78a94597f4f19fea633851fa2'. Error: The request was canceled due to the configured HttpClient.Timeout of 100 seconds elapsing.`
- And just like @jmglsn, we're getting super slow transfer speeds when checking out repositories in Actions as well.

**Comment by overbit** [comment]
Same in our repositories and even worst speed then @jmglsn report now:
![image](https://user-images.githubusercontent.com/2861984/211606906-5fe91af8-c944-4859-8841-562e1dd27db6.png)

**Reply by urica12** [reply]
> Same in our repositories and even worst speed then @jmglsn report now: ![image](https://user-images.githubusercontent.com/2861984/211606906-5fe91af8-c944-4859-8841-562e1dd27db6.png)

**Comment by joelnotified** [comment, marked as answer]
There has been an update to the GitHub status page: https://www.githubstatus.com/incidents/00km0kwpz23b

I haven't been able to confirm if that has resolved our issues yet though.

**Reply by jmglsn** [reply]
Yes it seems was exactly this one, thanks for reference :+1:

**Comment by urica12** [comment]
https://github.com/orgs/community/discussions/43834#discussioncomment-4647294

**Comment by urica12** [comment]
Report answered
